enterprise/server/backends/userdb/userdb.go

```diff
@@ -550,6 +550,9 @@ func (d *UserDB) CreateDefaultGroup(ctx context.Context) error {
 				if err := tx.Create(&c.group).Error; err != nil {
 					return err
 				}
+				if api_config.APIEnabled() {
+					c.apiKeyValue = newAPIKeyToken()
+				}
```

Comment:
The logic here looks a little funky since we also set the apiKeyValue on line 545. Do we need this block?

Reply:
I see what you mean. Fixed!